### PR TITLE
Use consistent formatting in API docs for `forest_blocks`

### LINKED ISSUE #1673: Enable `cargo-spellcheck` for `forest_blocks`
**Issue summary**

Make sure that `cargo spellcheck` can run without errors in `./blockchain/blocks/`. For details, see the tracking issue.

**Other information and links**

### PR DESCRIPTION
**Pull request by lemmih**
**Summary of changes**
Changes introduced in this pull request:
- Mark all non-dictionary words as code.

**Reference issue to close (if applicable)**

Closes #1673 

**Other information and links**

### CODE REVIEW

File: .config/forest.dic [hunk 1]
```diff
@@ -1,6 +1,11 @@
 10
 JWT
 JSON
+tipset/M
```
Comment (LesnyRumcajs) [inline .config/forest.dic:4]: what does `/M` mean?

**Review by LesnyRumcajs — APPROVED** [review]

File: .config/forest.dic (hunk 1 above)
Comment (lemmih) [reply .config/forest.dic:4]: It means that it'll accept both `tipset` and `tipsets`.

File: .config/forest.dic [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+bls
```
[the remark this answers is not in the document]
Comment (lemmih) [reply .config/forest.dic]: It's case-sensitive. I've updated as per your suggestions.

[the remark this answers is not in the document]
Comment (LesnyRumcajs) [reply .config/forest.dic]: thanks

**Review by tyshko5 — APPROVED** [review]

**Pull request merged**